liquidity/liquidity.go

```diff
@@ -506,16 +521,6 @@ func (m *Manager) SuggestSwaps(ctx context.Context, autoloop bool) (
 		return nil, ErrNoRules
 	}
 
-	// If our start date is in the future, we interpret this as meaning that
-	// we should start using our budget at this date. This means that we
-	// have no budget for the present, so we just return.
-	if m.params.AutoFeeStartDate.After(m.cfg.Clock.Now()) {
```

Comment:
I'm not sure I that the start date should be replaced with the budget, IMO they serve different purposes.

Reply:
You mean the budget timestamp?